src/components/scenes/TransactionsExportScene.tsx

```diff
@@ -164,28 +164,32 @@ class TransactionsExportSceneComponent extends React.PureComponent<
     })
   }
 
-  async componentDidMount(): Promise<void> {
-    try {
-      const { sourceWallet, tokenId } = this.props.route.params
-      const { disklet } = sourceWallet
-      const result = await disklet.getText(EXPORT_TX_INFO_FILE)
-      const exportTxInfoMap = asExportTxInfoMap(JSON.parse(result))
-      const tokenCurrencyCode =
-        tokenId ?? sourceWallet.currencyInfo.currencyCode
-
-      const { isExportBitwave, isExportCsv, isExportQbo } =
-        exportTxInfoMap[tokenCurrencyCode]
-
-      this.setState({
-        isExportBitwave,
-        isExportCsv,
-        isExportQbo
+  componentDidMount(): void {
+    const { sourceWallet, tokenId } = this.props.route.params
+    const { disklet } = sourceWallet
+    disklet
+      .getText(EXPORT_TX_INFO_FILE)
```

Comment:
This change is so big, what if we did a `loadInfoFile` helper function, and then do `componentDidMount() { this.loadInfoFile().catch(... }`